src/neuroglancer/segmentation_user_layer.ts

```diff
@@ -183,6 +201,13 @@ export class SegmentationUserLayer extends UserLayer {
     if (segmentEquivalences.size > 0) {
       x['equivalences'] = segmentEquivalences.toJSON();
     }
+    let {clipBounds} = this.displayState;
+    if (clipBounds.value) {
+      x['clipBounds'] = JSON.stringify({
```

Comment:
These toJSON() methods are actually supposed to return a JSON value as a javascript value, not the JSON string encoding.